aiida/engine/processes/calcjobs/manager.py

```diff
@@ -70,10 +90,9 @@ def get_last_updated(self):
 
     @gen.coroutine
     def _get_jobs_from_scheduler(self):
-        """
-        Get the current jobs list from the scheduler
+        """Get the current jobs list from the scheduler.
 
-        :return: A dictionary of {job_id: job info}
+        :return: a dictionary of {job_id: job info}
```

Comment:
is the "job info" of a particular format?
either put a string or some info about the format